composer.json

```diff
@@ -1,63 +1,26 @@
 {
     "name": "chubbyphp/chubbyphp-socket-server-mock",
-    "description": "Chubbyphp Socket Server Mock",
+    "description": "Fork of Chubbyphp Socket Server Mock",
     "keywords": ["chubbyphp", "slim", "socket", "server", "mock"],
     "license": "MIT",
-    "authors": [
-        {
-            "name": "Dominik Zogg",
-            "email": "dominik.zogg@gmail.com"
-        }
-    ],
     "require": {
-        "php": "^7.2",
-        "symfony/process": "^3.4.43|^4.4.11|^5.0"
+        "php": "^8.0",
```

Comment:
shouldn't this be `^8.0|^7.2` or `^8.0|^7.4` ? that way we could also use it with our php projects which aren't on 8 yet, although i think it's only used on media-api. would be worth checking.

Reply:
As I see it's only used in media-api, and I don't see it used in other project in the future. And even if so, we'll hopefully have them running on php 8 by then, so I would rather keep it more strict.